docs/guide/setup/creating_atom_mappings_and_scores.rst

```diff
@@ -64,8 +58,7 @@ In an interactive notebook we can view a 2D representation of the mapping.
 In this view,
 atoms that are deleted are coloured red, while atoms that undergo an elemental transformation are coloured blue.
 Similarly, bonds that are deleted are coloured red,
-while bonds that change, either bond order or are between different elements,
-are coloured blue.
+while bonds that change (either bond order change or element change), are coloured blue.
```

Comment:
As far as I remember I think it's possible to have blue colored elements when atoms are placed too far apart comparing the state A with the state B, not sure if we want to talk about this case since it may add unnecessary complexity, but I thought it could be worth thinking about it at this point.

Reply:
@ijpulidos you might be thinking of perses 😅 (caveat - I only really remember the first iteration that Mike and I worked on, I don't remember the code doing this, it might have changed!)